src/App.js

```diff
@@ -26,40 +28,44 @@ const App = () => {
   };
 
   return (
-    <form onSubmit={checkUserAnswers}>
-      {data.map(question => {
-        return (
-          <fieldset key={question.id}>
-            <legend>{question.text}</legend>
-
-            {question.answers.map(answer => {
-              return (
-                <div key={answer.id}>
-                  <input
-                    type="radio"
-                    id={`${question.id}.${answer.id}`}
-                    name={question.id}
-                    value={answer.id}
-                    onClick={handleAnswerSelect}
-                  />
-                  <label htmlFor={`${question.id}.${answer.id}`}>
-                    {answer.text}
-                  </label>
-                </div>
-              );
-            })}
-          </fieldset>
-        );
-      })}
-      {Object.keys(userAnswers).length === data.length && (
-        <button>Submit</button>
-      )}
-      {shouldDisplayScore && (
-        <h2>
-          You have got {score} out of {data.length}
-        </h2>
-      )}
-    </form>
+    <Container>
+      <Row>
+        <Col> <h1 style={{ textAlign: "center" }} >CYF Quiz</h1>  </Col>
+      </Row>
```

Comment:
Using columns only makes sense if you are positioning multiple elements horizontally. Here, we only have one element `h1` so we don't need `Row` or `Col` since it doesn't do anything for us.

Also, adding `style` to html is generally not good practise. Let's remove it and we don't need to centre it just now

Reply:
I will change that, thanks.